content/installation/java/Java-linux-packages.md

```diff
@@ -0,0 +1,157 @@
+
+## Ubuntu-Based Systems
+
+### Install the Java agent
+
+To install the Contrast Java agent for Ubuntu-based systems, complete the following steps.
+
+* Configure your system to Contrast's distribution agnostic Debian repository.
+
+```
+curl https://contrastsecurity.jfrog.io/contrastsecurity/api/gpg/key/public | sudo apt-key add -
+echo "deb https://contrastsecurity.jfrog.io/contrastsecurity/debian-staging/ all contrast" | sudo tee /etc/apt/sources.list.d/contrast-staging-all.list
+```
+
+* Once you've finished configuration, install the Contrast Java agent.
+
+```
+sudo apt-get update && sudo apt-get install contrast-java-agent
+```
+
+* Edit the */etc/contrast/contrast-java-config.yaml* file to configure Contrast Java agent to connect to the Contrast application. <!-- (Go to the YAML [Configuration doc](_________) for more info.) -->
+
+### Next steps
+
+At this stage, you may either complete the agent installation process by follow the instructions for the [Java application container](installation-javainstall.html) of your choice, or continue on to **Add the Exec Helper** package.
+
+### Add the Exec Helper
+
+The Exec Helper package for the Contrast Java agent supports Ubuntu LTS distributions Trusty, Xenial and Bionic. Complete the following steps to install the package.
+
+* Before you install the package, use the following command to verify that you have already installed the Contrast Java agent.
+
+```
+dpkg -s contrast-java-agent
+```
+
+* If it isn't installed, follow the instructions in the **Ubuntu-Based Systems** section to install and configure the `contrast-java-agent` package.
+
+* To install the Exec Helper from Contrast's Debian repository, configure your system to use the repository tailored for your Ubuntu LTS distribution.
+
+```
+curl https://contrastsecurity.jfrog.io/contrastsecurity/api/gpg/key/public | sudo apt-key add -
+echo "deb https://contrastsecurity.jfrog.io/contrastsecurity/debian-staging/ $(bash -c '. /etc/lsb-release && echo $DISTRIB_CODENAME') contrast" | sudo tee /etc/apt/sources.list.d/contrast-staging-$(bash -c '. /etc/lsb-release && echo $DISTRIB_CODENAME').list
+```
+
+* Once you've finished configuration, install the `contrast-java-agent-exec-helper` package.
+
+```
+sudo apt-get update && sudo apt-get install contrast-java-agent-exec-helper
+```
+
+* Verify that the Contrast Java Exec Helper is installed by executing `java` and confirming that Contrast starts by observing console messages.
+
+```
+$ bash -c "java -Dcontrast.stdout=true -version 2>1 | grep Contrast | head -n 1"
+[Contrast] Wed Aug 15 17:37:23 UTC 2018 No TeamServer configuration detected. Agent will only be reporting to local listeners (e.g., Eclipse Plugin).
+```
+
+* The Exec Helper affects all new shells, as indicated by the `bash -c` in the verification command. Consequently, it will be present in your current shell. Reload your shell to enable the Exec Helper. Restart any `java` services to enable Contrast for those services.
```

Comment:
Should this read
> it will **not** be present in your current shell

Which is why you should reload your shell to enable it?

Reply:
Yes, good catch